packages/tinacms/src/toolkit/fields/plugins/date-format.ts

```diff
@@ -4,6 +4,39 @@ import type { DatetimepickerProps } from 'react-datetime';
 export const DEFAULT_DATE_DISPLAY_FORMAT = 'MMM DD, YYYY';
 export const DEFAULT_TIME_DISPLAY_FORMAT = 'h:mm A';
 
+/**
+ * Gets the user's locale-specific date format using abbreviated months
+ * Returns formats like "MMM DD, YYYY" for US or "DD MMM, YYYY" for UK, etc.
+ * Falls back to the hardcoded default format if locale detection fails
+ */
+export const getLocaleDateFormat = (): string => {
```

Comment:
Since this component uses `moment`. 

hmm instead of this can we use something like the following:

```tsx
import moment from 'moment';
...
export const getLocaleDateFormat = (): string => {
  try {

    const locale =
      Intl.DateTimeFormat().resolvedOptions().locale;

    moment.locale(locale);

    return moment.localeData().longDateFormat('ll');
 	} catch (error) {
    return DEFAULT_DATE_DISPLAY_FORMAT;
  }
```